ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/router_isis.schema.yml

```diff
@@ -44,6 +44,140 @@ keys:
+                      - str
+                    description: Number of seconds.
+      authentication:
+        type: dict
+        keys:
+          key_id:
+            type: dict
+            keys:
+              id:
+                type: int
+                convert_types:
+                  - str
+                min: 0
+                max: 65535
+                description: Configure authentication keyid.
+              algorithm:
+                type: str
+                valid_values:
+                  - sha-1
+                  - sha-224
+                  - sha-256
+                  - sha-384
+                  - sha-512
+              key:
+                type: str
+                convert_types:
+                  - int
+                valid_values:
+                  - '0'
+                  - '7'
+                  - '8a'
+                description: Configure authentication key.
+              password:
+                type: str
+              rfc_5310:
+                type: dict
+                keys:
+                  key:
+                    type: str
+                    convert_types:
+                      - int
+                    valid_values:
+                      - '0'
+                      - '7'
+                      - '8a'
+                    description: Configure authentication key.
+                  password:
+                    type: str
+                  enabled:
+                    type: bool
+                  level_1:
+                    type: bool
+                  level_2:
+                    type: bool
+          key:
+            type: str
+            convert_types:
+              - int
+            valid_values:
+              - '0'
+              - '7'
+              - '8a'
+            description: Configure authentication key.
+          password:
+            type: str
+          mode:
+            type: dict
+            keys:
+              md5:
+                type: dict
+                keys:
+                  enabled:
+                    type: bool
+                    description: Configure authentication at level-1 and level-2 both.
+                  level_1:
+                    type: bool
+                    description: Configure authentication at level-1.
+                  level_2:
+                    type: bool
+                    description: Configure authentication at level-2.
+                  rx_disabled:
+                    type: dict
+                    keys:
+                      enabled:
+                        type: bool
+                        description: Disable authentication on receive side at level-1 and level-2 both.
+                      level_1:
+                        type: bool
+                        description: Disable authentication on receive side at level-1.
+                      level_2:
+                        type: bool
+                        description: Disable authentication on receive side at level-2.
```

Comment:
Maybe we can change these to two values like:
```
levels: <str; "both", "level-1", "level-2"; required>
rx_disabled_levels: <str; "both", "level-1", "level-2">
```

Reply:
Would it make sense instead to keep levels as required and add a boolean for rx_disabled? or can I have both with and without rx_disabled at the same time?

Reply:
Fixed